CODEOWNERS

```diff
@@ -2,4 +2,11 @@
 # Each line is a file pattern followed by one or more owners.
 
 # These owners will be the default owners for everything in the repo.
-*       @ajaysrivas @shaileshsvc @NeeravChoudhari
+*       @ajaysrivas @shaileshsvc
+
+# These owners will be the code owners for the following subdirectories in the repo.
+/bvt/               @Jyoti-cs
```

Comment:
Why is it reporting error (could be false) for user names, is it some utf-8 text copy or something?

Reply:
it is because some of the users don't have write access to the repository, [CORTX-33898](https://jts.seagate.com/browse/CORTX-33898) is created for the same.

Reply:
Ok resolved.